### PR TITLE
Fixes various performance issues

### PR DESCRIPTION
**Pull request by Nash0x7E2**
This PR fixes various performance issues across the codebase. Some of the core areas touched by this PR are:
- Removed two-column list view implementation in favor of a lazy list 
- Fixes hero animation in collective preview 
- Removes animated crossfade on the list (both lists were being constructed on top of each other )
- Clean up `expression_open` and add size to the expression previews 
- Resolve layout switching issue 
- Changed audio expression image preview to use cached network image; prevents multiple network calls

### CODE REVIEW

File: lib/widgets/previews/expression_preview/two_column_preview/two_column_expression_preview_types/audio.dart [hunk 1]
```diff
@@ -173,8 +175,9 @@ class AudioPreviewWithPhoto extends StatelessWidget {
             Colors.black38,
             BlendMode.srcOver,
           ),
-          image: NetworkImage(
+          image: CachedNetworkImageProvider(
```
Comment (sunyatax) [inline lib/widgets/previews/expression_preview/two_column_preview/two_column_expression_preview_types/audio.dart]: Already added CachedNetworkImageProvider in all relevant places in dev

**Comment by sunyatax** [comment]
Nice work on the performance ops - feels a lot better.

See my comments re: various parts of the app that are no longer consistent with the previous/desired UX.

**Comment by Nash0x7E2** [comment]
> Nice work on the performance ops - feels a lot better.
> 
> See my comments re: various parts of the app that are no longer consistent with the previous/desired UX.

Thanks, which parts are you referring too?

**Comment by sunyatax** [comment]
@Nash0x7E2 I think one solution for optimizing the SingleColumnSliverListView performance is changing the SliverChildListDelegated in collective_populated_list.dart to [SliverChildBuilderDelegate](https://api.flutter.dev/flutter/widgets/SliverChildBuilderDelegate-class.html)

**Comment by Nash0x7E2** [comment]
Yes, I was looking at it but we are doing some stuff with adding an extra Sizedbox to the list. Playing around with this now in my branch

**Pull request merged**